### PR TITLE
RE2022-246: Add ngram based string matching

### PR DESCRIPTION
**Pull request by MrCreosote**
... and apply it to the classification field

### CODE REVIEW

File: src/common/collection_column_specs/genome_attribs-GTDB.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ columns:
 
    - key: classification  # GTDB classification, might need better tokenizing
      type: string
-     filter_strategy: fulltext
```
Comment (Tianhao-Gu) [inline src/common/collection_column_specs/genome_attribs-GTDB.yml:52]: Do we need to update genome_attribs-GROW/ENIGMA.yml as well?

Comment (MrCreosote) [reply src/common/collection_column_specs/genome_attribs-GTDB.yml:52]: ENIGMA and GROW are just softlinks to PMI

Comment (Tianhao-Gu) [reply src/common/collection_column_specs/genome_attribs-GTDB.yml:52]: I see.

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**